static/templates/admin/editAccessSurrogateForm.html

```diff
@@ -0,0 +1,19 @@
+<iframe id="upload_file_frame" name="upload_file_frame" height="0" width="0" style="display: none;"></iframe>
+<form id="create_simple_object_form" method="post" action="/services/api/edit/accessSurrogate/<%= pid %>" enctype="multipart/form-data" target="upload_file_container" class="edit_form create_simple_object_form">
+        <% if (metadata.datastream.findIndex(d => d.startsWith('access_surrogate')) !== -1) { %>
```

Comment:
Probably unindent this block once so it lines up with all the other immediate children of the form